build/pkgs/semigroups/distros/repology.txt

```diff
@@ -0,0 +1 @@
+gap
```

Comment:
repology has a specific `gap-semigroups` category. Please add arch too: the package name is `gap-packages`